builtin/logical/pki/path_roles.go

```diff
@@ -273,23 +302,30 @@ or revoked, so this option is recommended only for certificates that are
 non-sensitive, or extremely short-lived. This option implies a value of "false"
 for "generate_lease".`,
 			},
+
 			"require_cn": &framework.FieldSchema{
-				Type:        framework.TypeBool,
-				Default:     true,
-				Description: `If set to false, makes the 'common_name' field optional while generating a certificate.`,
+				Type:         framework.TypeBool,
+				Default:      true,
+				Description:  `If set to false, makes the 'common_name' field optional while generating a certificate.`,
+				DisplayName:  "Use CSR Common Name",
+				DisplayValue: true,
```

Comment:
This can now be removed.

Reply:
I don't know how I missed these. removed a bunch more from pki